v3/lint/registration.go

```diff
@@ -313,7 +379,7 @@ var globalRegistry = NewRegistry()
 // registration process.
 //
 // IMPORTANT: RegisterLint will panic if given a nil lint, or a lint with a nil
-// Lint pointer, or if the lint's Initialize function errors, or if the lint
+// Lint pointer, or if the lint's Initialize function errors, or if the lint`
```

Comment:
typo: stray backtick
```suggestion
// Lint pointer, or if the lint's Initialize function errors, or if the lint
```